src/base/bittorrent/session.cpp

```diff
@@ -155,6 +155,19 @@ namespace
 
         return expanded;
     }
+
+    QStringList findAllFiles(const QString &folderPath)
+    {
+        QStringList files;
```

Comment:
BTW, what to do with symbolic links? Another question: if a file is not readable or writable, do we need it?